CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project will be documented in this file.
 - A new generic `read` method on `UVCal` that, like the `read` methods on our other
 objects, supports all file types and a new `from_file` class method to allow one-line
 reading of UVCal files.
+- A new `UVData.new()` method (based on new function `new_uvdata`) that creates a new,
+self-consistent `UVData` object from scratch from a set of flexible input parameters.
```

Comment:
This should be at the top (reverse chronology)

Reply:
I think this got screwed up in the rebase. Fixed now.